proposals/1984_ca-bundle-from-secret-ref/README.md

```diff
@@ -11,124 +11,261 @@ Once approved and implemented, they become historical documents.
 If you are reading an old proposal, please be aware that the
 features described herein might have continued to evolve since.
 
-# CA Bundles from secret refs
+# Source Certificate Authority Data from Secrets and ConfigMaps into Pinniped Custom Resources
 
 ## Problem Statement
 
 Many Pinniped custom resources (CRs) have an inline `certificateAuthorityData` to specify a base-64 encoded CA Bundle.
-These fields require manual intervention to set and update.
-Customers who use Kubernetes-native certificate-management tooling (such as `cert-manager`) may wish to keep their
-CA bundles in a secret that their tooling can manage.
+Instead of providing the CA bundles inline, users/cluster-operators should be able to pass these as a reference to either kubernetes secrets or configmaps.
```

Comment:
This sentance is describing the proposed solution, but this section is intended to introduce the problem that we are trying to solve (not the solution). In other words, *why* do we think we need this new feature?

Reply:
See https://github.com/vmware-tanzu/pinniped/blob/main/proposals/README.md for outline of proposal document format.

Reply:
Your user story section is basically the problem statement, I think.

Reply:
Removed the user story section and updated the problem statement.